distribution/environments/aws-ec2-cloud/CloudFormation/cloudformation-template.json

```diff
@@ -53,7 +53,7 @@
               "export DOCKER_IMAGE=jenkins/evergreen:aws-ec2-cloud\n",
               "echo \"", { "Ref": "PrivateKey"}, "\" > ssh-agents-private-key\n",
               "docker pull $DOCKER_IMAGE\n",
-              "docker run -d -p 8080:8080 -p 50000:50000 ",
```

Comment:
This will prevent any JNLP-based agents from being launchable from Jenkins Essentials, such as Windows agents.

I don't think that's a desirable outcome, regardless of how undesirable Windows agents may be :smiley_cat: 


I think restricting 50000 to the security group makes more sense than removing this entirely